app/src/main/java/org/eyeseetea/malariacare/data/remote/api/UserAttributesRemoteDataSource.java

```diff
@@ -0,0 +1,81 @@
+package org.eyeseetea.malariacare.data.remote.api;
+
+import android.util.Log;
+
+import com.fasterxml.jackson.databind.JsonNode;
+
+import org.eyeseetea.malariacare.data.database.model.UserDB;
+import org.eyeseetea.malariacare.domain.boundary.IUserAttributesRemoteDataSource;
+import org.eyeseetea.malariacare.domain.entity.Credentials;
+import org.eyeseetea.malariacare.domain.entity.UserAttributes;
+import org.eyeseetea.malariacare.domain.exception.PullUserAttributesException;
+
+public class UserAttributesRemoteDataSource extends OkHttpClientDataSource implements IUserAttributesRemoteDataSource {
```

Comment:
Logic in this class should be in a class like:
public class UserAccountAPIDataSource implements IUserAccountDataSource